nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/util/FTPTransfer.java

```diff
@@ -304,6 +304,12 @@ public void flush() throws IOException {
     }
 
     @Override
+    public void flush(final FlowFile flowFile) throws IOException {
+        final FTPClient client = getClient(flowFile);
+        client.completePendingCommand();
```

Comment:
or at least in this methods case return that boolean and explain the same meaning so the caller can decide case by case how/if to handle.  Good catch